### PR TITLE
feat(typography): add paragraphs and headings - 1271

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Add missing paragraph sizes and headings, according to [the specifications](https://next-v2--europa-component-library.netlify.com/ec/guidelines/typography/)

Also use rem for line-heights

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 7b2ebada033a123a303a99d2898fde8382bccd56

https://deploy-preview-938--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-tag/ec-component-tag.scss [hunk 1]
```diff
@@ -13,8 +13,9 @@
   .ecl-tag {
     align-items: center;
     background-color: map-get($ecl-colors, 'grey-15');
-    border-radius: (2 * $ecl-spacing-xs + $ecl-font-size-s * $ecl-line-height-s) /
-      2;
+    border-radius: (
+        2 * $ecl-spacing-xs + ($ecl-font-size-s * $ecl-line-height-s) / 1rem
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-tag/ec-component-tag.scss]:
```suggestion
        2 * $ecl-spacing-xs + $ecl-line-height-s
```

the line-height is now the right value (we don't need to take the font-size into account)

**Review by yhuard — CHANGES_REQUESTED** [review]
A small change is requested (both EC and EU).

File: src/systems/eu/implementations/vanilla/packages/eu-component-tag/eu-component-tag.scss [hunk 2]
```diff
@@ -13,8 +13,9 @@
   .ecl-tag {
     align-items: center;
     background-color: map-get($ecl-colors, 'grey-15');
-    border-radius: (2 * $ecl-spacing-xs + $ecl-font-size-s * $ecl-line-height-s) /
-      2;
+    border-radius: (
+        2 * $ecl-spacing-xs + ($ecl-font-size-s * $ecl-line-height-s) / 1rem
```
Comment (yhuard) [inline src/systems/eu/implementations/vanilla/packages/eu-component-tag/eu-component-tag.scss]: please fix this one too

File: src/systems/eu/implementations/react/packages/typography/stories/Index.jsx [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
-import React from 'react';
+import React, { Fragment } from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import { withKnobs, text } from '@storybook/addon-knobs';
 
+import demoContentHeading from '@ecl/eu-specs-typography/demo/data--heading';
 import demoContentParagraph from '@ecl/eu-specs-typography/demo/data--paragraph';
 
 storiesOf('Typography', module)
```
Comment (yhuard) [inline src/systems/eu/implementations/react/packages/typography/stories/Index.jsx]: can you prefix it with "Utilities"? i.e. `storiesOf('Utilities/Typography', module)`
it makes it easier to find stories (note: we should do it for other stories as well)

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/specs/utilities/typography/docs/code.mdx [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
 
+import demoContentHeading from '../demo/data--heading';
 import demoContentParagraph from '../demo/data--paragraph';
 
-## Default paragraph
+## Headings
+
+<Playground
+  system="ec"
+  selectedKind="Typography"
```
Comment (yhuard) [inline src/systems/ec/specs/utilities/typography/docs/code.mdx]: please update the `selectedKind` -> `Utilities/Typography` wherever it's needed

**Review by yhuard — DISMISSED** [review]

**Review by degliwe — APPROVED** [review]

**Pull request merged**